src/components/generic/OptionsList/OptionsList.scss

```diff
@@ -17,6 +17,7 @@
       align-items: center;
       cursor: pointer;
       color: var(--wxc-text-color);
+      border: 1px solid transparent;
```

Comment:
you can remove this pixel from the vertical padding to preserve the total height pf the element